\section{Introduction}

  \subsection{Histones contribute to nucleosome dynamics}

<<<<<<< HEAD
    The chromatin packaging of eukaryote genomes performs the functions of
    compacting very large lengths of DNA into the microscopic cell nucleus,
    facilitating chromosomal movements during cell division,
    and acting as a substrate for molecular mechanisms acting on the genome.

    The building block of eukaryotic chromatin structure is the nucleosome, 
    comprising \SI{147}{\bp} of DNA wrapped around an octamer of two copies each
    of core histones H2A, H2B, H3 and H4 \citep{luger1997crystal}.
    Nucleosome core particles are arranged in a linear chain separated by DNA linkers,
    and can be further compacted into higher order chromatin structures.

    Chromatin functions at both the molecular and cellular levels 
    requires the capability for dynamic rearrangement.
    Chromatin structure can be modulated through nucleosomes
    by changing the arrangement of histones and DNA in a process known as remodelling \addref{?},
    or by altering histone chemical composition 
    through post-translational modification \addref{?} or exchange of histone variants \addref{?}.

    A large amount of evidence has been accumulated about 
    the static structure of the nucleosome at atomic resolution \addref{?},
    and about the static arrangement of polymeric chromatin \addref{?}.
    However, the mechanisms for dynamic rearrangement of chromatin
    are much less well understood or integrated between the molecular and polymer levels \addref{?}.

  \subsection{Nucleosome dynamics and histone SIN mutants}

    The archetype of ATP-dependent nucleosome remodelling enzymes is the SWI/SNF complex,
    which was identified in screens for mating type SWItching \citep{SWI-mutants}
    and Sucrose Non Fermentation \citep{SNF-mutants-original-discovery, SNF-mutants2}
    in \textit{Saccharomyces cerevisiae}.

    Mutations were subsequently identified that compensate for the loss of the SWI/SNF complex
    that are collectively known as SIN mutations because they provide SWI/SNF INdependence \addref{?}.
    A subset of SIN mutants are single amino-acid changes in core histones H3 and H4,
    providing a direct link between SWI/SNF and chromatin.
    This also suggests the mutated residues in the histone proteins 
    influence the same pathways for nucleosome dynamics leveraged by the enzyme 
    and that these residues are significant for nucleosome stability.

    This predicted that the stability of SIN mutant containing nucleosomes would be affected in chromatin.
    The hypothesis that nucleosomes containing histone SIN mutants
    have altered stability has been tested \textit{in vitro}
    It was observed that SIN mutant containing nucleosomes display higher 
    thermally driven nucleosome sliding mobility \citep{flaus2004sin}
    and that the mutated residues compromise histone-DNA contacts in crystal structures \addref{muthurajan2004crystal}.
=======
    The chromatin packaging of eukaryote genomes performs the
    functions of compacting very large lengths of DNA into the
    microscopic cell nucleus, facilitating chromosomal movements
    during cell division, and acting as a substrate for molecular
    mechanisms acting on the genome.

    The building block of eukaryotic chromatin structure is the
    nucleosome, comprising \SI{147}{\bp} of DNA wrapped around an
    octamer of two copies each of core histones H2A, H2B, H3 and H4
    \citep{luger1997crystal}.  Nucleosome core particles are arranged
    in a linear chain separated by DNA linkers, and can be further
    compacted into higher order chromatin structures.

    Chromatin functions at both the molecular and cellular levels
    requires the capability for dynamic rearrangement.  Chromatin
    structure can be modulated through nucleosomes by changing the
    arrangement of histones and DNA in a process known as remodelling
    \addref{?}, or by altering histone chemical composition through
    post-translational modification \addref{?} or exchange of histone
    variants \addref{?}.

    A large amount of evidence has been accumulated about the static
    structure of the nucleosome at atomic resolution \addref{?}, and
    about the static arrangement of polymeric chromatin \addref{?}.
    However, the mechanisms for dynamic rearrangement of chromatin are
    much less well understood or integrated between the molecular and
    polymer levels \addref{?}.

  \subsection{Nucleosome dynamics and histone SIN mutants}

    The archetype of ATP-dependent nucleosome remodelling enzymes is
    the SWI/SNF complex, which was identified in screens for mating
    type SWItching \citep{SWI-mutants} and Sucrose Non Fermentation
    \citep{SNF-mutants-original-discovery, SNF-mutants2} in
    \textit{Saccharomyces cerevisiae}.

    Mutations were subsequently identified that compensate for the
    loss of the SWI/SNF complex that are collectively known as SIN
    mutations because they provide SWI/SNF INdependence
    \addref{kruger1995amino}.  A subset of SIN mutants are single
    amino-acid changes in core histones H3 and H4, providing a direct
    link between SWI/SNF and chromatin.  This also suggests the
    mutated residues in the histone proteins influence the same
    pathways for nucleosome dynamics leveraged by the enzyme and that
    these residues are significant for nucleosome stability.

    This predicted that the stability of SIN mutant containing
    nucleosomes would be affected in chromatin.
    The hypothesis has been tested \textit{in vitro}
    where it was observed that SIN mutant nucleosomes display higher 
    thermally driven nucleosome sliding mobility \citep{flaus2004sin}
    and that the mutated residues compromise histone-DNA contacts in
    crystal structures \citep{muthurajan2004crystal}.
>>>>>>> origin/master

    Although histone protein SIN mutants affect nucleosomes \textit{in vitro} and in \textit{S. cerevisiae},
    their effect on nucleosomes and chromatin in the more complex environment 
    of mammailian nuclei has not been demonstrated.
    Observing this effect would validate the functional significance of the residues
    and contribute to explaining the high conservation of histone protein sequences across eukaryotes.

    Although histone protein SIN mutants affect nucleosomes \textit{in
      vitro} and in \textit{S. cerevisiae}, their effect on
    nucleosomes and chromatin in the more complex environment of
    mammailian nuclei has not been demonstrated.  Observing this
    effect would validate the functional significance of the residues
    and contribute to explaining the high conservation of histone
    protein sequences across eukaryotes.

  \subsection{Fluorescence Recovery After Photobleaching}

    Fluorescence Recovery After Photobleaching (FRAP) is an optical technique
    that reports the dynamics of fluorescently tagged molecules within live cells.
    Tagged molecules inside a small region are irreversibly photobleached by
    a high power focused laser beam and the recovery rate of fluorescence
    in the bleached area is measured. The recovery rate is interpreted as unbleached molecules
    from outside the region at the time of photobleaching diffusing into the bleached area.
    It is assumed that this fluorescence recovery reflects natural protein movement.

    FRAP can be described by a simple chemical equilibrium:

    \begin{displaymath}
      F + S \overset{k_{on}}{\underset{k_{off}}{\rightleftharpoons}} FS
    \end{displaymath}

<<<<<<< HEAD
    where $F$ represents freely diffusing proteins, 
    $S$ represents immobile vacant binding sites, 
=======
    where $F$ represents freely diffusing proteins,
    $S$ represents immobile vacant binding sites,
>>>>>>> origin/master
    and $FS$ the complex between the two when the proteins are bound to the sites. 
    The value of \Kon{} and \Koff{},
    are estimated from the rate at which photobleached $F$ is replaced in the $FS$ complex.

    Ongoing development of FRAP has led to increasingly complex models
    that are both more precise and accurate than simple inverse exponential decay \addref{?}.
    Despite their sophistication, these models require assumptions
    that are difficult to maintain over long experimental observation times.
    Firstly, equilibrium must be maintained throughout the entire experiment 
    so that both \Kon{} and \Koff{} remain constant.
    This also requires that concentrations of both $F$ and $S$ remain constant.
    Secondly, distribution of the fluorescently tagged molecule must mimic the endogenous protein.
    And finally, the binding sites must be part of a large and relatively immobile complex
    on the time and length scale of the recovery.

<<<<<<< HEAD
  \subsection{FRAP measurements of chromatin proteins}

    FRAP has been extensively used to obtain qualitative and quantitative insight 
    into the kinetic properties of non-histone chromatin-associated factors 
    (Phair and Misteli, 2000b; Essers et al., 2005a; Agresti et al., 2005). 
    These rely on the reasonable assumption that chromatin is 
    relatively immobile in the interphase nuclei (Abney et al., 1997)
    since the factors show recovery on the scale of seconds to minutes indicating high mobility.
    H2Bâ€“GFP (Kanda et al., 1998) has become the standard reference 
    for the immobile fraction in such FRAP experiments (Dey et al., 2000).

    The dynamics of core histones was measured by FRAP 
    in a seminal and widely cited study by \citet{KimuraCook}.
    Multiple H2B populations were delineated with distinctive exchange rates.
    Some 3\% of H2B had a rapid recovery within minutes, 
    40\% had slow recovery with a t1/2 of 130 min, 
    and the majority of H2B molecules had a very slow recovery with t1/2 of over 8 hours
    that was considered to be effectvely immobile.

    In contrast to H2B as a histone dimer component,
    the tetramer histones H3 and H4 were found to have even slower mobility.
    There was no rapid populations, with only slow and very slow populations 
    of 16-22\% and 62-68\% being identified.
    
    In combination with additional heterokaryon data, Kimura and Cook 
    interpreted the rapid, slow and very slow exchanging H2B populations 
    as correlating with transcription units, euchromatin and heterochromatin respectively.
    They assigned over 80\% of H3 and H4 as immobile, 
    whereas the remaining \textasciitilde20\% was suggested to be mobilised by remodelling.
    This latter small but significant fraction of histones should provide an 
    opportunity to observe the dynamics of tetramer histones in live mammalian cells.
=======
  \subsection{FRAP measurements of histones}

    FRAP has been extensively used to obtain qualitative and
    quantitative insight on the kinetic properties of chromatin bound
    proteins \citep{phair2000high, essers2005nuclear, agresti2005gr}.
    These rely on the established assumption that chromatin is
    relatively immobile in the interphase nuclei
    \citep{abney1997chromatin} since the factors show recovery
    on the scale of seconds to minutes indicating high mobility.
    H2B--GFP \citep{KevinH2BGFP} has become
    the standard reference for immobile fraction in such
    FRAP experiments \citep{dey2000bromodomain}.

    %% Histones are often used as the reference for immobile in FRAP.
    %% Of special interest is Dey et al 2000 ``A Bromodomain Protein,
    %% MCAP, Associates with Mitotic Chromosomes and Affects G2-to-M
    %% Transition'', which Kimura and Cook use as an example of FRAP
    %% done on histones, but actually, they used H2B-GFP as an example
    %% that showed no FRAP recovery after 100 minutes.  From Dey et
    %% all:
    %%
    %%     "As shown in Fig. 6B, GFP-MCAP fluorescence was reduced to
    %%      background levels immediately after photobleaching but
    %%      recovered ~86% of its intensity within 1 min. By contrast,
    %%      histone H2B-GFP did not recover any fluorescence over this
    %%      period. These results indicate that while histone H2B, a
    %%      stable component of chromatin is immobile, [... talk about
    %%      MCAP]"

    The dynamics of core histones was measured by FRAP in a seminal
    and widely cited study by \citep{KimuraCook}.  Multiple H2B
    populations were delineated with distinctive exchange rates.  Some
    \pcent{3} of H2B had a rapid recovery within minutes, \pcent{40}
    had slow recovery with a \halflife[] of \SI{130}{\minute}, and the
    majority of H2B molecules had a very slow recovery with \halflife[] of
    over 8 hours that was considered to be effectively immobile.

    In contrast to H2B as a histone dimer component, the tetramer
    histones H3 and H4 were found to have even slower mobility.  There
    was no rapid populations, with only slow and very slow populations
    of \SIrange{16}{22}{\percent} and \SIrange{62}{68}{\percent} being
    identified.

    In combination with additional heterokaryon data, Kimura and Cook
    interpreted the rapid, slow and very slow exchanging H2B
    populations as correlating with transcription units, euchromatin
    and heterochromatin respectively.  They assigned over \pcent{80}
    of H3 and H4 as immobile, whereas the remaining
    \SI{\approx 20}{\percent} was suggested to be mobilised by
    remodelling.  This latter small but significant 
    slowly exchanging fraction of histones 
    should provide an opportunity to observe the dynamics of
    tetramer histones in live mammalian cells.

>>>>>>> origin/master

  \subsection{Aims and objectives}

    In order to test the implications for nucleosome structure and function
<<<<<<< HEAD
    of metazoan H4~R45H that exhibited the highest increase 
    in nucleosome mobility \textit{in vitro} of all the histone SIN mutants,
    we set out to determine the exchange characteristics of this histone by FRAP.
    In attempting to achieve quantitative measurements 
    we addressed multiple technical challenges associated with 
    measuring subtle kinetic alterations of nucleosome dynamics over long time periods in live cells. 
    This allowed us to define the limitations of FRAP in observing molecules 
=======
    of metazoan H4~R45H that exhibited the highest increase
    in nucleosome mobility \textit{in vitro} of all the histone SIN mutants,
    we set out to determine the exchange characteristics
    of this histone by FRAP.
    In attempting to achieve quantitative measurements
    we addressed multiple technical challenges associated with
    measuring subtle kinetic alterations of nucleosome dynamics
    over long time periods in live cells.
    This allowed us to define the limitations of FRAP in observing molecules
>>>>>>> origin/master
    such as core histones with extremely slow exchange rates.
