\section{Results}

    To investigate the challenges of performing FRAP in mammalian cells over several hours,
    we transfected HeLa cells with a H2B-EGFP under the control of a CMV promoter
    and observed recovery in cells over 8 hours \frefp{fig:kill-frap:cell-movement}.
    
    \begin{figure}
      \centering
      \missingfigure{FRAP of HeLa cells expressing H2B--EGFP}
      \captionIntro{FRAP of HeLa cells expressing H2B--EGFP.}
                   {HeLa cells were transfected with pBOS--H2B-EGFP
                   with circle FRAP performed in a widefield microscope over 8~hours.
                   Pre-bleach image at top left followed by image sequence at 20~min intervals.}
      \label{fig:kill-frap:cell-movement}
    \end{figure}

    The images taken at 20 min intervals revealed considerable changes
    in position of the nucleus, arrangement of features within the nucleus, 
    and the shape of the bleach spot \frefp{fig:kill-frap:cell-movement}.
    Initial attempts to quantiatively analyse the images
    did not yield meaningful estimates of H2B exchange kinetics (data not shown).

    The central requirement of FRAP is to accurately identify and 
    quantitate the signal in the photobleached spot over time.
    This led us to pursue both cell biological approaches to minimise movement 
    and computational approaches to track imaged regions.

    %% Could we show panel 1B as a plot of 
    %% the aspect ratio of the bleach spot and cell over time 
    %% as an illustration of the changes.

  \subsection{Inhibition of cell motility}

    We first attempted to reduce motility by restricting the space available
    using cells at higher confluency for FRAP experiments.
    This was performed using a HeLa cell line stably expressing 
    the H4~R45H SIN mutant to ensure even H4 fluorescence in all cells.
    It resulted in some decrease in movement of cells
    but did not achieve not a complete immobilisation \frefp{fig:kill-frap:confluent-hela}.
    In fact, nuclei frequently underwent considerable reshaping as cells 
    apparently squeezed between their neighbours.

      \begin{figure}
        \centering
        \includegraphics[width=\textwidth]{results/confluent-hela.png}
        \captionIntro{Movement of confluent HeLa cells during a FRAP experiment}
          {
            HeLa-derived stable cell line expressing H4~R45H-YFP
            with half-nuclear FRAP performed in a confocal microscope over 8~hours.
            Pre-bleach image at top left followed by image sequence at 21~min intervals.
          }
        \label{fig:kill-frap:confluent-hela}
      \end{figure}

    %% Could we show panel 2B as a plot of 
    %% the deviation of the cell or nuclear centroid in X and Y 
    %% relative to the first frame as a representative example
    %% for the HeLa cells in the first and second figures.
    %% This needs to be in absolute micron units independent of the magnification.
    %% We could even show the distribution of aspect ratio deviations 
    %% for a sample of individual cells as a box plot as panel 2C

    Many mammalian primary cells display the property of contact inhibition.
    In this cellular growth response, cells enter senescence and reduce motion
    when surrounded by neighbours \citep{?}.

\todo{Should we be saying horse fibroblasts or horse cells?}
    Since HeLa cells have lost the ability to activate contact inhibition \citep{?},
    we obtained an immortalised primary horse fibroblast cell line known to display this property.
    Because primary cell line transfections typically have much lower efficiency
    and confluent cells have lower expression with each cell division,
    we transfected cells at \SI{70}{\percent} confluence and performed FRAP after 3 days.

    Despite reaching confluence where contact inhibition was expected, 
    we continued to observe movement of the transfected horse cells \frefp{fig:kill-frap:confluent-horse}.
    The characteristics of cell movement also differ dramatically between primary horse and HeLa cells,
    The horse cell nuclei exhibit a helical motion about the vector of their movement 
    in $x$ and $y$ axes \frefp{fig:kill-frap:confluent-horse}
    whereas HeLa nucelar motion was mostly restricted to the $z$ axis relative to the dish \frefp{fig:kill-frap:cell-movement}.

    \begin{figure}
      \centering
      \includegraphics[width=\textwidth]{results/confluent-horse.png}
      \captionIntro{Movement of confluent primary horse cells during a FRAP experiment}
        {
          Immortalised horse cells were transfected with pBOS--H2B-EGFP
          with circle FRAP performed in a widefield microscope over 8~hours.
          Pre-bleach image at top left followed by image sequence at 15~min intervals.
        }
      \label{fig:kill-frap:confluent-horse}
    \end{figure}

    \subsubsection{Tracking of cell movement}

      As an alternative strategy, we implemented cell tracking
      in order to transform images into a common frame using
      consecutive image cropping and image registration.
      This approach was implemented as a program named CropReg.

      \begin{figure}
        \centering
        \includegraphics[width=\textwidth]{results/cropreg.png}
        %% imaging was done every 10 minutes, but we are skipping
        %% every other panel
        \captionIntro{Automatic tracking and alignment of moving cells}
          {
            Using CropReg, we successfully tracked individual cells during
            a time-series microscope experiment. The top left corner of each
            panel displays the tracked and aligned cell. Imaging was performed
            in a widefield microscope. Time interval between panels 20~minutes.
            Cells are a stable line derived from HeLa, expressing H3 tagged
            with YFP.
          }
        \label{fig:kill-frap:cropreg}
      \end{figure}

      Using this image processing approach we were able to track individual cell nuclei
      throughout an entire sequence of FRAP experiments \frefp{fig:kill-frap:cropreg} provided
      that nuclei did not overlap.
      Although only a small minority of image sequences satisfied this requirement,
      it was possible to collect sufficient observations for FRAP calcultions.

  \subsection{Chromatin movement}

    While performing the FRAP experiments, we observed some movement
    within the cell nuclei. These could not be accounted for simple rotational
    movement around the $x$ or $y$ axis, and resembled more the movement
    of individual bodies within the nuclei.

    \subsubsection{Selection of \G1{} cells}
      %% There's no chemical equilibrium in S phase

      A possible cause of this chromatin movement comes from changes in
      the cell cycle phase. During the S~phase, the DNA is replicated,
      doubling the content of the chromatin.
      More importantly, this breaks
      a core assumption of FRAP, that the system remains in equilibrium
      during the entire experiment. This does not hold if the DNA, the
      binding sites for our model, duplicate in number.

      If the FRAP experiments can't be performed during S~phase and
      mitosis, we are limited to \G1{} and \G2{}. Considering
      the length of the HeLa cell cycle and the requirements to image
      for a time period of 8~hours, we are further limited to \G1{}.
      In addition, the FRAP experiment must be performed early in
      \G1{}~phase to avoid crossing over to the S~phase.

      %% The only reason this was required was because the LSM 510
      %% that we were using could not make Z stack and time lapse
      %% at the same time.
%      \begin{figure}
%        \centering
%        \missingfigure{Hela cells splitting}
%        \captionIntro{Picking cells at early G$_1$.}
%                     {We imaged cells that were entering mitosis and picked their
%                      daughter cells for the FRAP experiments. Because HeLa cells lift
%                      away from the dish during mitosis, opening the
%                      pinhole and set the Z-center in between the cell dividing plane
%                      and dish bottom was necessary. Ends up nothing being properly in focus but we
%                      can track things fine. Of course, some cells still floated away.}
%        %% TODO explicit parameters
%        \label{fig:kill-frap:picking-early-g1}
%      \end{figure}

      To do this, cells in mitosis were selected and tracked during 4~hours.
      After this time period, we used the daughter cells which we could be
      confident of being in early \G1{}.
      %% we also waited some 2 hours after mitosis since that's when cells
      %% unpack their chromosomes.

      During mitosis, HeLa cells form a sphere slightly above the plane of
      other cells, and keep a weak connection to the growth surface.
      Because of this, they easily detach, which is the basis for the
      mitotic shake-off method, and float away from the field of vision
      which requires a larger number
      of initial selected cells. In addition, to minimize any effect that
      may arise from imaging, it was done at minimal laser power and every
      30~minutes, just enough to allow manual tracking.
      Finally, since our system did not permit simultaneous Z-stack and time
      lapse imaging, and cells in mitosis are in a separate focal plane,
      imaging was performed with the pinhole sized to the max and focused
      in between the two planes. While this
      created very blurred images, it allowed to visualize all cells during
      the entire procedure.

      However, even after selecting cells in this cell cycle, movement within
      the bleach spot could still be observed.

    \subsubsection{Inverse FRAP}

      Due to the non-homogeneous nature of the chromatin, it was difficult
      to assess the total extent of the observed movement. To
      better visualize this, we performed inverse FRAP which allows us
      to track the movement of the bleach spot only.

      For this purpose, we replaced the EGFP tag in our H2B plasmid
      with photoactivatable GFP (PAGFP), a GFP derivative that requires
      activation by a specific wavelength to become fluorescent. This
      allows us to activate a specific spot of the nucleus and visualize
      its movement.

      Since PAGFP cannot be easily detected before photoactivation, cells
      were co-transfected with mCherry--\textalpha--tubulin which localises
      exclusively to the cytoplasm, giving an outline of the nuclear region
      \frefp{fig:kill-frap:ifrap}.

      \begin{figure}
        \centering
        \subbottom[pre-activation]{
          \includegraphics[width=0.45\textwidth]
          {results/ifrap-pre.png}
          \label{fig:kill-frap:ifrap-pre}
        }
        \hfill
        \subbottom[post-activation]{
          \includegraphics[width=0.45\textwidth]
          {results/ifrap-post.png}
          \label{fig:kill-frap:ifrap-post}
        }
        \subbottom[activated spot over time]{
          \includegraphics[width=\textwidth]
          {results/ifrap.png}
          \label{fig:kill-frap:ifrap-timeframe}
        }
        \captionIntro{Inverse FRAP experiment showing chromatin movement}
          {
            HeLa cells co-transfected with mCherry--\textalpha--tubulin and
            H2B type1-J tagged with PAGFP.
            \subcaptionref{fig:kill-frap:ifrap-pre} The cell nucleus, target
            for photoactivation, can be easily identified as the ``empty''
            region via the mCherry channel on which would otherwise be an
            invisible feature on the GFP channel;
            \subcaptionref{fig:kill-frap:ifrap-post} spot after activation;
            \subcaptionref{fig:kill-frap:ifrap-timeframe} detail of the
            activated spot every 20~minutes. Rather than a gradual loss of
            fluorescence that maintains the circular shape, the activated spot
            kind of unfolds itself spreading the region of interest.
          }
        \label{fig:kill-frap:ifrap}
      \end{figure}

      Using this FRAP variant, the movement of chromatin was more noticeable.
      Rather than an homogeneous loss of fluorescence, the activated
      spot uncurled itself overtime with individual branches of
      localized PAGFP appearing in the nuclei \frefp{fig:kill-frap:ifrap}.
